shake256-avx2/.gitignore

```diff
@@ -3,4 +3,5 @@ test/*
 PQCsignKAT_*.rsp
 PQCsignKAT_*.req
 PQCgenKAT_sign
-keccak4x/KeccakP-1600-times4-SIMD256.o
\ No newline at end of file
+keccak4x/KeccakP-1600-times4-SIMD256.o
+*.s
```

Comment:
with the file having `.S` (capital letter) now this probably also needs to be updated?

Reply:
We wouldn't want to ignore the file itself, right?  The `.s` is generated form the `.S` when compiling.

Reply:
ah you're not running `peachpy` in Make of course.

Reply:
Indeed.  Try to keep dependencies to compile to a minimum.